### ISSUE TITLE
Raidbots - Top Gear Simulation - Legendary Talent Ring Issue

### ISSUE THREAD

**Issue by Kendwynn** [seriallos/raidbots-issues#31]
I think there is a bug where the top gear simulation have trouble recognizing the legendary talent ring in certain specializations.

Whenever I run a top gear simulation with my hunter in survival specialization with the Serpent Sting talent selected (the talent the legendary ring gives me as passive), I can select either Butchery or Dragonfire Granade talent on the ring itself down below. However, If I do the same thing the same hunter in Beast Mastery specialization I don't get to choose what talent I want to sim with.

**Comment by seriallos** [comment]
Can you provide SimC input where this happens? Are you certain the input is using the talent provided by the ring?

**Comment by Kendwynn** [comment]
[BM SimC Input.txt](https://github.com/seriallos/raidbots-issues/files/1356356/BM.SimC.Input.txt)
[Survival SimC Input.txt](https://github.com/seriallos/raidbots-issues/files/1356357/Survival.SimC.Input.txt)

I've tested it now just to be sure. 
When I use the Survival SimC Input with passive talent the ring gives selected in-game I get to choose between Butchery and Dragonfire Granade on the ring itself down below.

When I use the BM SimC input with the passive talent the ring gives selected in-game I don't get to choose between One with the Pack or Blink Strike on the ring itself donw below. 

I've also tried to to refresh the page, switch between Top Gear and Gear Compare, manually change talent in the string itself pasted on raidbots etc with no luck.
